Meshtastic/Helpers/BLEManager.swift

```diff
@@ -942,7 +946,11 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 									}
 								}
 								hopNodes.append(traceRouteHop)
-								routeBackString += "\(hopNode?.user?.longName ?? (node == 4294967295 ? "Repeater" : String(hopNode?.num.toHex() ?? "unknown".localized))) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized)dB) --> "
+
+								let hopName = hopNode?.user?.longName ?? String(hopNode?.num.toHex() ?? "unknown".localized)
```

Comment:
was the `node == 4294967295` was previously hardcoded here. Any idea why that was the case? (yay if we fixed a hardcoding bug)

Reply:
The logic identifying routers needs to be in there, why did it get removed?

Reply:
It's not necessarily a router (or repeater). It can be a node that can't decrypt because it doesn't have the channel key, or in the route back it's likely a node on firmware before 2.5.

The only case that could cause a repeater to show as `0xFFFFFFFF` as well is if it was set to `ALL_SKIP_DECODING`.

Reply:
Could say repeater (no key) or something but it needs to be in there.  Most users are just using the app and are not digging through the docs.

Reply:
Why not just "unknown"? There are multiple options (old firmware is another), and we don't which one it is.

If it's a repeater not set to `ALL_SKIP_DECODING`, it will be its NodeNum, not `0xFFFFFFFF`.

Reply:
Ah, or will it show `0xFFFFFFFF` now when it doesn't know the long name? That would indeed be confusing.

Reply:
Thoughts on next steps here?
It sounds like adding back special handling for 0xFFFFFFFF makes sense. But should it call that out as 'unknown' since a hop could come back with that set for multiple reasons? Or is something more precise that these entries should be named?

Reply:
`unknown` feels like a clean answer if we cannot say who they are for any reason

Reply:
> Thoughts on next steps here? It sounds like adding back special handling for 0xFFFFFFFF makes sense. But should it call that out as 'unknown' since a hop could come back with that set for multiple reasons? Or is something more precise that these entries should be named?

Yes I want the special handling in there

Reply:
Should the hopName end up as "unknown" or something else in this path?

Reply:
> Should the hopName end up as "unknown" or something else in this path?

Repeater like it was.

Reply:
done